static/routing-tree.js

```diff
@@ -167,11 +168,13 @@ function massage(root) {
 
   root.matchers = matchers;
 
+  root.receiver_configs = getReceiverConfigs(root.receiver, receivers);
```

Comment:
```suggestion
  root.receiverConfig = getReceiverConfig(root.receiver, receivers);
```

(singular since it's just one receiver config per node, and camelCase for style consistency)